src/util/getList.js

```diff
@@ -9,14 +9,21 @@ module.exports = id => {
     const list = { ...match };
 
     // Load full features
-    list.features = featuresData.map(feature => ({
-        ...feature,
-        value: list.features.includes(feature.id) ? 1 : 0,
-    })).sort((a, b) => {
+    list.features = featuresData.map(feature => {
+        delete feature['$schema'];
+
+        return {
+            ...feature,
+            value: list.features.includes(feature.id) ? 1 : 0,
+        };
```

Comment:
```suggestion
        const withValue = {
            ...feature,
            value: list.features.includes(feature.id) ? 1 : 0,
        };
        
        delete withValue['$schema'];
        
        return withValue;
```
Just a nitpick so we aren't mutating the original object, only the shallow clone